src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/ipc.yaml

```diff
@@ -2,9 +2,8 @@
 system:
   certificateFilePath: "/greengrass/v2/device.pem.crt"
   privateKeyPath: "/greengrass/v2/private.pem.key"
-  rootCaPath: "/greengrass/v2/AmazonRootCA1.pem"
-  rootpath: "/greengrass/v2"
   ipcSocketPath: "/ipcFolder/ipc.socket"
+  rootpath: "/greengrass/v2"
```

Comment:
why do we need this change?

Reply:
It isn't needed and removing it ensures we are running from our own temp dir instead of a shared space that may have anything in it.

Reply:
Removing root ca path prevents exceptions in MQTT client because the ca doesn't exist.